LOG.tbasicu

```diff
@@ -27,22 +27,23 @@ uses "File"
 #Region "Equates"
 
 ' Log equates
-
-%LT_Debug = 1 as Long 'Display only if isDebug = %TRUE
-%LT_Time = 2 as Long ' Prepend DateTime$
-%LT_ToConsole = 4 as Long
-%LT_ToConsoleTitle = 8 as Long
-%LT_ToMsgBox = 16 as Long
-%LT_ToLogFile= 32 as long
-%LT_ToStatusBar= 64 as long
-'%LT_ToRichedit = 128 as Long ?
+begin Const
+  %LT_Debug          = 1 'Display only if isDebug = %TRUE
+  %LT_Time           = 2 ' Prepend DateTime$
+  %LT_ToConsole      = 4
+  %LT_ToConsoleTitle = 8
+  %LT_ToMsgBox       = 16
+  %LT_ToLogFile      = 32
+  %LT_ToStatusBar    = 64
+  '%LT_ToRichedit = 128
+end const
 
 $FILE_ROTATION_START_MESSAGE = "Log file rotation * START *."
```

Comment:
Feel free to reject this, but it is easier to read the texts if they start at the same offset.